packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -182,4 +204,36 @@ export class BaseButton extends BaseComponent<IButtonProps, {}> implements IButt
         null
       );
   }
+
+  protected _onRenderMenuIcon = (props: IButtonProps): JSX.Element => {
```

Comment:
Is autobind preferred over a lambda? I thought we were trying to be consistent by moving over to the lambda :) I'm good wither way. I think @dzearing was steering me towards lambdas.